src/library/scala/math/package.scala

```diff
@@ -315,13 +356,83 @@ package object math {
   /** Returns the size of an ulp of the given `Double` value.
     * @group ulp
     */
-  def ulp(x: Double): Double = java.lang.Math.ulp(x)
+  def ulp(d: Double): Double = java.lang.Math.ulp(d)
```

Comment:
In fact this change has to be reverted or extended with a deprecated parameter name to be binary compatible. To maintain the focus in this change to the addition of missing method defer this renaming to an separate PR.